### PR TITLE
NOISSUE - Clean up error handling, remove unwrap() calls, logging

### PR DESCRIPTION
**Pull request by jmcconnell26**
meaningful messages

### CODE REVIEW

**Comment by anderejd** [comment]
Sorry for the delay, it seems I missed the notification. I would like add you as maintainer if you are interested? @tarcieri would also need to approve.

I'll review this PR now.

File: cargo-geiger/src/mapping.rs [hunk 1]
```diff
@@ -32,18 +34,11 @@ pub trait GetLicenceFromCargoMetadataPackageId {
...
         &self,
         package_id: &cargo_metadata::PackageId,
-    ) -> Option<String>;
-}
-
-pub trait GetPackageVersionFromCargoMetadataPackageId {
-    fn get_package_version_from_cargo_metadata_package_id(
-        &self,
-        package_id: &cargo_metadata::PackageId,
-    ) -> Option<cargo_metadata::Version>;
```
Comment (anderejd) [inline cargo-geiger/src/mapping.rs:46]: Nice simplification. 👍

**Comment by anderejd** [comment]
The new error messages look great!

**Comment by jmcconnell26** [comment]
Hi @anderejd, I'd definitely be keen to be added as a maintainer!

**Comment by anderejd** [comment]
@tarcieri Since this project now exists under rust-secure-code it seems reasonable to me to have you on board with adding another maintainer. Your thoughts?

**Comment by anderejd** [comment]
@tarcieri Ping

**Comment by anderejd** [comment]
@jmcconnell26 Invite has been sent!

**Comment by jmcconnell26** [comment]
Awesome, accepted it now!

**Pull request merged**